kube-runtime/src/controller/mod.rs

```diff
@@ -423,6 +427,7 @@ where
 #[derive(Clone, Debug, Default)]
 pub struct Config {
     debounce: Duration,
+    concurrency: usize,
```

Comment:
just curious, why not keep it as a usize, since rust will automatically infer the size of the integer based on the architecture of the host machine?

Reply:
yep, in general that is a good reason to use usize. I mostly like explicit sizes on public interfaces like this because it lets me know what the maximum allowed value actually is (without asking about architectures).